docs/source/installation.rst

```diff
@@ -157,26 +157,102 @@ error log (in Apache on Debian/Ubuntu, for example, this is ``/var/log/apache2/e
 --------------------
 Deployment on Docker
 --------------------
-
 It is also possible to deploy the server using Docker.
 
-First build the Docker Image.  From the ``deploy`` folder run:
+First, you need an environment running the docker daemon. For non-production use, we recommend `boot2docker <http://boot2docker.io/>`_. For production use you should install docker on a stable linux distro.
+`Platform specific Docker installation instructions <https://docs.docker.com/installation/) (OS X and Windows are instructions for boot2docker>`_
+
+**Local Dataset Mounted as Volume**
+
+If you already have a dataset on your machine, you can download and deploy the apache server in one command: 
 
 .. code-block:: bash
 
-  $ docker build .
+  $ docker run -e GA4GH_DATA_SOURCE=/data -v /my/ga4gh_data/:/data:ro -itd -p 8000:80 --name ga4gh_server afirth/ga4gh_apache_server:prod
```

Comment:
Can you explain what's happening here please? I'm sorry, I'm a complete docker noob, so I'm struggling to understand how  the config files and the environment variables are fitting together.